servatrice/src/serversocketinterface.cpp

```diff
@@ -164,14 +164,24 @@ void ServerSocketInterface::readClient()
             newCommandContainer.ParseFromArray(inputBuffer.data(), messageLength);
         }
         catch(std::exception &e) {
-            qDebug() << "Caught std::exception in" << __FILE__ << __LINE__ << __PRETTY_FUNCTION__;
+            qDebug() << "Caught std::exception in" << __FILE__ << __LINE__ << 
+#ifdef _MSC_VER // Visual Studio
+                __FUNCTION__;
+#else
+                __PRETTY_FUNCTION__;
+#endif
```

Comment:
IIRC there are other places where protobuf's methods gets called (the replay/deck transfer code); all these calls could throw exceptions, and we should be able to catch them.
So yes, it might gets used somewhere else, but within the whole snippet of code implementing exception handling. Maybe we could reimplement this as a external helper.. i'll think a bit on it.